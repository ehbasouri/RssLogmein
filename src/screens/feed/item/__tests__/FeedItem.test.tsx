import {fireEvent, render} from '@testing-library/react-native';
import React from 'react';
import {FeedItem} from '..';

describe('Feed Item tests', () => {
  it('render feed item and user action tests', () => {
    let props: any = {
      onPress: jest.fn(),
      feed: {},
    };
    const {toJSON, getByTestId} = render(<FeedItem {...props} />);

    const item = getByTestId('feed-item');
    fireEvent.press(item);
    expect(props.onPress).toHaveBeenCalled();
    expect(toJSON).toMatchSnapshot();
  });
});
