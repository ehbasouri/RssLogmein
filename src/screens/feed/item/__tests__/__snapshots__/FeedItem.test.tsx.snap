// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feed Item tests render feed item and user action tests 1`] = `[Function]`;
